x-pack/legacy/plugins/uptime/public/pages/overview.tsx

```diff
@@ -84,17 +84,13 @@ export const OverviewPageComponent = ({ autocomplete, setBreadcrumbs, indexPatte
         </EuiFlexGroup>
         <EuiSpacer size="s" />
         <StatusPanel
-          absoluteDateRangeStart={absoluteDateRangeStart}
```

Comment:
I'm not sure why the linter isn't picking this up, but we can remove `absoluteStart/End` declarations further up in this function.

Example: `const params = getUrlParams();`

Reply:
Done that.